### ISSUE TITLE
Game can't handle more than 140 saves

### ISSUE THREAD

**Issue by UltraStars3000** [AliveTeam/alive_reversing#110]
**OG Bug**
If there are 141 saves or more, you won't be able to load the latest ones as they won't be displayed by the game.
Note that you can still save, the .sav file will still be made, but it won't display neither, unless you remove at least one.

**Comment by mouzedrift** [comment]
imagine having 141 saves lmao

**Comment by paulsapps** [comment]
I'll have 141 after 100 more bugs are raised 🥇

**Comment by UltraStars3000** [comment]
**UPDATE:**
This isn't due to the number of saves, but more to the number of chars for all the saves combined. Where saves with numbers works perfectly, the ones with letters no, the game seems to no be able to display the most recent saves with letters once you go over an "overall char limit".
Issue name edited!

**Comment by paulsapps** [comment]
Makes sense, if you put any file in the game dir with a long name it will crash the game.
